### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by Habtamu210**
### Project description:
- foundations for the Bookstore website. 
- Created the user interface using React.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,177 @@
...
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- <a href="#" target="_blank">Live Demo Link</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Project Presentation Video <a name="live-demo"></a>
+
+- <a href=""</a>
```
Comment (DammyShittu) [inline README.md]: - [ ] Kindly remove this section from your readme since you have no video presentation.

File: src/components/book.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+          </div>
+          <div>
+            <button type="submit">Add Book</button>
+          </div>
+        </form>
+      </div>
+
+    </>
+  );
+}
```
Comment (DammyShittu) [inline src/components/book.js]: - [ ] This component should split into two. The first component should simply take in `title` and `author` as props (as you have correctly done). Its responsibility should solely be the rendering of ONE book. It should be a reusable component.

- [ ] The second component should be a form component. Kindly remove the form in this component and move it into a new component whose sole responsibility would be to add books.

- [ ] Finally, create another component (name can be `BookList`) that would render this Book component and the Form component.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @Habtamu210,

## Highlights
✔️ Descriptive PR
✔️ Passing linters
✔️ Good branch name

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @Habtamu210 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Used Git flow ✔️
- Used react-router ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**